src/index.spec.ts

```diff
@@ -1282,6 +1282,8 @@ describe('EppoPrecomputedJSClient E2E test', () => {
   });
 
   it('logs assignments correctly', () => {
+    const mockAssignmentCache = td.object<AssignmentCache>();
+    globalClient.useCustomAssignmentCache(mockAssignmentCache);
```

Comment:
Another case of needing to reset the assignment cache (filled by other tests running) to get logging to work correctly in this test specifically